certify-mock-identity.properties

```diff
@@ -118,12 +127,99 @@ mosip.certify.key-values={\
+                          'phone': { 'display': {{'name': 'Phone Number', 'locale': 'en' }}},\
+                          'dateOfBirth': { 'display': {{'name': 'DOB', 'locale': 'en' }}},\
+                          'gender': { 'display': {{'name': 'Gender', 'locale': 'en' }}},\
+                          'email': { 'display': {{'name': 'Email Id', 'locale': 'en' }}},\
+                          'region': { 'display': {{'name': 'Region', 'locale': 'en' }}},\
+                          'province': { 'display': {{'name': 'Province', 'locale': 'en' }}},\
+                          'UIN': { 'display': {{'name': 'UIN', 'locale': 'en' }}},\
+                          'VID': { 'display': {{'name': 'VID', 'locale': 'en' }}},\
+                          'postalCode': { 'display': {{'name': 'Postal Code', 'locale': 'en' }}}\
+                     }},\
+                    'display': {{'name': 'Mock Verifiable Credential', \
+                                  'locale': 'en', \
+                                  'logo': {'url': 'https://${mosip.api.public.host}/inji/mosip-logo.png','alt_text': 'a square logo of a MOSIP'},\
+                                  'background_color': '#f8efea',\
+                                  'background_image': { 'uri': 'https://${mosip.api.public.host}/inji/mosip-logo.png' }, \
+                                  'text_color': '#000000'}},\
+                    'order' : {'fullName','phone','dateOfBirth','gender','email','region','province','UIN', 'VID', 'postalCode'}\
+                 },\
+                 "MockVerifiableCredential_mdoc":{\
+                      'format': 'mso_mdoc',\
+                      'doctype': 'org.iso.18013.5.1.mDL',\
+                      'scope' : 'sample_vc_mdoc',\
+                      'cryptographic_binding_methods_supported': {'cose_key'},\
+                      'credential_signing_alg_values_supported': {'ES256'},\
+                      'proof_types_supported': {'jwt': {'proof_signing_alg_values_supported': {'ES256'}}},\
+                      'claims': {\
+                          'org.iso.18013.5.1': {\
+                              'given_name': {'display': {{'name': 'Given Name','locale': 'en'}}},\
+                              'family_name': {'display': {{'name': 'Family Name','locale': 'en'}}},\
+                              'issue_date': {'display': {{'name': 'Issue Date','locale': 'en'}}},\
+                              'expiry_date': {'display': {{'name': 'Expiry Date','locale': 'en'}}},\
+                              'birth_date': {'display': {{'name': 'Birth Date','locale': 'en'}}},\
+                              'issuing_country': {'display': {{'name': 'Issuing Country','locale': 'en'}}},\
+                              'document_number': {'display': {{'name': 'Document Number','locale': 'en'}}}\
+                       }},\
+                      'display': {{'name': 'Mobile Driving License', \
```

Comment:
Mobile Driving license ??

Reply:
This was added as a part of mdoc being added in the wellknown. Should this be removed?

Reply:
If a new plugin file is created to avoid the inconsistency between what ldp vc and mdoc data, then this needs to be removed, otherwise this is fine

Reply:
There is no new plugin for mdoc. It has been added in mock-certify-plugin and respective properties.

Reply:
I have added one comment in the digital credential plugin PR, so marking this as resolved